unittests/NodeAPI/CMakeLists.txt

```diff
@@ -97,15 +109,27 @@ target_include_directories(node_lite
     ../../API/hermes_node_api/node_api
 )
 
-set(NodeApiTestsSources
-  child_process.cpp
-  child_process.h
-  string_utils.cpp
-  string_utils.h
-  test_main.cpp
-  test_main.h
-  test_basics.cpp
-)
+if(APPLE)
+  set(NodeApiTestsSources
+    child_process_mac.cpp
+    child_process.h
+    string_utils.cpp
+    string_utils.h
+    test_main.cpp
+    test_main.h
+    test_basics.cpp
+  )
+else()
+  set(NodeApiTestsSources
+    child_process.cpp
+    child_process.h
+    string_utils.cpp
+    string_utils.h
+    test_main.cpp
+    test_main.h
+    test_basics.cpp
+  )
+endif()
```

Comment:
Perhaps you could build up the list of files which are shared between unit tests and `node-lite` and reuse that across these two targets?

Reply:
Fixed in `reduce redundancy in list of sources`, thanks!